analytics/pubxai/processor/processor_test.go

```diff
@@ -0,0 +1,214 @@
+package processor
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/prebid/openrtb/v20/openrtb2"
+	utils "github.com/prebid/prebid-server/v3/analytics/pubxai/utils"
+	utilsMock "github.com/prebid/prebid-server/v3/analytics/pubxai/utils/mocks"
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProcessLogData_NilAuctionObject(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockUtilsService := utilsMock.NewMockUtilsService(ctrl)
+	processorService := &ProcessorServiceImpl{
+		utilService: mockUtilsService,
+	}
+
+	ao := (*utils.LogObject)(nil)
+	auctionBids, winningBids := processorService.ProcessLogData(ao)
+	assert.Nil(t, auctionBids)
+	assert.Nil(t, winningBids)
+}
```

Comment:
Kindly reformat to Go table driven tests.

Reply:
updated all the tests to Go table driven tests.